deepcell_toolbox/metrics.py

```diff
@@ -188,8 +188,8 @@ def __init__(self,
         self.y_true = y_true
         self.y_pred = y_pred
 
-        self.n_true = len(np.unique(self.y_true)) - 1
-        self.n_pred = len(np.unique(self.y_pred)) - 1
+        self.n_true = len(np.unique(self.y_true[self.y_true > 0]))
+        self.n_pred = len(np.unique(self.y_pred[self.y_pred > 0]))
```

Comment:
I think its true for us that we will never have a negative integer, though I think excluding 0 is a safer option:

```suggestion
        self.n_true = len(np.unique(self.y_true[np.non_zero(self.y_true)]))
        self.n_pred = len(np.unique(self.y_pred[np.non_zero(self._y_pred)]))
```

Reply:
Yes, if we start getting negative labels we've got bigger problems